### DISCUSSION TITLE
vscode: add `/UserMode` to install vscode with user installer

### DISCUSSION THREAD

**Discussion by soredake** [chocolatey-community/chocolatey-packages#2040] [Ideas]
Vscode installed with system installer (default in choco package) cannout silently update itself because of UAC. It would be nice to have `/UserMode` option to install with user installer (can be found here https://code.visualstudio.com/#alt-downloads) so vscode can update itself silently in background.

**Comment by soredake** [comment]
https://code.visualstudio.com/docs/setup/windows#_user-setup-versus-system-setup

**Comment by pauby** [comment]
This looks to be a different installer to the one used in the package? Depending on how this is done, it will need either a significant change to the existing package or a new package entirely.

**Reply by soredake** [reply]
>This looks to be a different installer to the one used in the package?

Yes, i'm fine with either a new -user package or package parameter.

**Reply by pauby** [reply]
@AdmiringWorm what are your thoughts on this?

**Reply by pascalberger** [reply]
I would suggest a separate package for user-level installer, like we already have for insider builds. Especially considering that there are limitation with Chocolatey for user-level installers (e.g. when used with Chocolatey Agent, on multi-user machines, etc)

**Reply by pauby** [reply]
@soredake if you want to add the package here (which would make sense) then please raise a PR.
